hooks/post_gen_project.py

```diff
@@ -114,7 +114,6 @@ def remove_gql_react_files():
     file_names = [
         join("client/src/utils", "mutations.ts"),
         join("client/src/utils", "queries.ts"),
-        join("client/src/utils", "get-cookie.js"),
```

Comment:
Is this used on the Vue side then?

Reply:
ahhh saw other comment

Reply:
Nope. That fn only runs post gen of a react project. So `client/src/utils` is from react